docs/arch/adr-33.md

```diff
@@ -0,0 +1,29 @@
+# ADR 33: Survey Task Feedback
+
+July 21, 2021
+
+## Context
+
+We plan to implement feedback for the survey task.
+
+Currently in PFE, the only feedback related to the survey task is for the Gravity Spy project, and is referred to as "Gravity Spy Gold Standard". Gravity Spy Gold Standard is a PFE experimental feature (per workflow configuration object) that checks a subject's metadata for `'#Type' === 'Gold'` and provides feedback after classification completion with the [`gs-gold-standard-summary`](https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/classifier/gs-gold-standard-summary.jsx). Survey task annotation values are compared to subject metadata `'#Label'` values. A message is then shown accordingly (i.e. success, failure, or special message per subject metadata `'#post_classification_feedback'`).
+
+## Decision
+
+To implement a survey task feedback strategy within [the existing feedback framework](https://github.com/zooniverse/front-end-monorepo/tree/master/packages/lib-classifier/src/store/feedback/strategies). The initial survey task annotation feedback will presume the annotation has one annotation value per survey task choice. The subject metadata will include:
```

Comment:
Let's have this be a simple feedback strategy for just comparing the string choices and make a note in the consequences section that there may be a request in the future for a more complex one that compares the choice questions too? This strategy will not implement this.